Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.py

```diff
@@ -451,17 +726,58 @@ def delete_comment_command(args: Dict[str, Any]) -> CommandResults:
     )
 
 
-def download_attachment_command(args: Dict[str, Any]) -> Any:
+def download_attachment_by_filename_command(args: Dict[str, Any]) -> dict:
+    case_id = args.get("case_id", None)
+    file_name = args.get("file_name", None)
```

Comment:
```suggestion
    case_id = args.get("case_id")
    file_name = args.get("file_name")
```

As `None` is the default default-value